<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snake</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icon.png</normaloff>:/icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="layout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHow_to_Play">
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="title">
     <string>How to play</string>
    </property>
    <addaction name="actionRules"/>
    <addaction name="actionControls"/>
   </widget>
   <widget class="QMenu" name="menuGame_mode">
    <property name="title">
     <string>Game mode</string>
    </property>
    <addaction name="actionGame_mode"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_this_app"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionRussian"/>
    <addaction name="actionUkrainian"/>
   </widget>
   <widget class="QMenu" name="menuHigh_score">
    <property name="title">
     <string>High score</string>
    </property>
    <addaction name="actionHigh_score"/>
   </widget>
   <addaction name="menuHow_to_Play"/>
   <addaction name="menuHigh_score"/>
   <addaction name="menuGame_mode"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionAbout_this_app">
   <property name="text">
    <string>About this app</string>
   </property>
   <property name="toolTip">
    <string>About this app</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="toolTip">
    <string>About Qt Framework</string>
   </property>
  </action>
  <action name="actionRules">
   <property name="text">
    <string>Rules</string>
   </property>
  </action>
  <action name="actionControls">
   <property name="text">
    <string>Controls</string>
   </property>
  </action>
  <action name="actionGame_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionUkrainian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ukrainian</string>
   </property>
  </action>
  <action name="actionHigh_score">
   <property name="text">
    <string>Show</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
